index.html

```diff
@@ -22,115 +22,127 @@
-                    <li class="nav-item">
-                        <a href="#">
-                            <img src="images/icons/Mail-icon.svg" alt="Mail icon" class="mail-icon">
-                        </a>
-                    </li>
-                </ul>
-            </div>
-        </nav>
-    </div>
-    <section class="headline">
-        <div class="headline-container">
-            <div>
-                <h1  class="primary-text">Hey There. <br>I'm Feven</h1>
-            </div>
-            <div>
-                <h2 class="supporting-text">I'am a Software Developer</h2>
-            </div>
-           <div>
-                <p class="description">I can help you build a product,feature or website look through some of my work and experience if you like what you see and have a project you need coded,don't hesitate to contact me.
-                </p>
-           </div>
-        </div>
-        <div class="social-media">
-            <ul class="social-icon">
-                <li class="icon">
-                    <a href="https://github.com/FevenSeyfu" class="icon-btn">
-                        <img src="images/icons/Github-icon.svg" alt="Github icon" class="bottom-icon">
-                    </a>
-                </li>
-                <li class="icon">
-                    <a href="https://www.linkedin.com/in/feven-seyfu-0008641b0/" class="icon-btn">
-                        <img src="images/icons/Linkedin-icon.svg" alt="linkedin icon" class="bottom-icon">
-                    </a>
-                </li>
-                <li class="icon">
-                    <a href="https://angel.co/u/feven-seyfu" class="icon-btn">
-                        <img src="images/icons/Angel list-icon.svg" alt="angelist icon" class="bottom-icon">
-                    </a>
-                </li>
-                <li class="icon">
-                    <a href="https://twitter.com/FevenSeyfu" class="icon-btn">
-                        <img src="images/icons/Twitter-icon.svg" alt="twitter icon" class="bottom-icon">
-                    </a>
-                </li>
-                <li class="icon">
-                    <a href="https://medium.com/@fevensey" class="icon-btn">
-                        <img src="images/icons/medium-icon.svg" alt="medium icon" class="bottom-icon">
-                    </a>
-                </li>
-            </ul>
-        </div>
-    </section>
-    <section class="works-section" id="#portfolio">
-
+		<div id="mobile-menu" class="">
+			<button class="cancel-btn" >
+				<img src="images/icons/Cancel-Icon.svg" alt="cancel button icon" class="cancel-icon">
+			</button>
+			<div class="main-navbar">
+				<ul class="nav-items mobile">
+					<li><a href="#portolio" class="nav-item mobile portfolio" >Portfolio</a></li>
+					<li><a href="#about" class="nav-item mobile about">About</a></li>
+					<li><a href="#contact" class="nav-item mobile contact">Contact</a></li>
+				</ul>
+			</div>
+		</div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Your Top elements(Nav elements) should be inside the `<header />` tag and your `Navigation` menu items should be inside the `<nav />` tag. Please implement my suggestions 🙏🏼 

```HTML
<header>
    <h1>Website Title</h1>
    <nav>
        <!-- Main Navigation inside <header> -->
        <ul>
            <li></li>
        </ul>
    </nav>
</header>
```

Reply:
Thank you for your feedback, I have amended the code and added the top section inside the header tag and I have also used the nav tag for the mobile menu.